src/elements/dialogs/session-details.html

```diff
@@ -210,6 +214,19 @@ <h3>{$ sessionDetails.speakers $}</h3>
       _getFeaturedSessionIcon(featuredSessions, sessionId) {
         return featuredSessions && featuredSessions[sessionId] ? 'bookmark-check' : 'bookmark-plus';
       }
+
+      _findSpeakers(speakers) {
```

Comment:
can be simplified to:
```
_findSpeakers(speakers) {
  if (speakers && typeof speakers[0] !== 'object') {
    return speakers.map((speakerId) => this.speakersMap[speakerId]);
  }
  return speakers;
}
```

Reply:
Done, even if I prefer the `fail fast` pattern which seems to be clearer for me.